Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -987,6 +984,19 @@ public final class SwiftModuleBuildDescription {
 
         return arguments
     }
+
+    /// Whether to build Swift code with whole module optimization (WMO)
+    /// enabled.
+    package var useWholeModuleOptimization: Bool {
+        if self.target.underlying.isEmbeddedSwiftTarget { return true }
```

Comment:
What does this impact if we decide we want to support debugging of embedded?

Reply:
The embedded compilation mode _requires_ `-wmo` to be passed to swiftc when in `-Onone` and with/without `-g` so it should be orthogonal. 

Without this particular patch, `swift build -c debug` does not work with embedded targets.

Reply:
Thanks!